packages/cli/src/util/projects/link.ts

```diff
@@ -95,11 +95,13 @@ async function getProjectLinkFromRepoLink(
   if (projects.length === 1) {
     project = projects[0];
   } else {
+    const selectableProjects =
+      projects.length > 0 ? projects : repoLink.repoConfig.projects;
     const { p } = await client.prompt({
       name: 'p',
       type: 'list',
       message: `Please select a Project:`,
-      choices: repoLink.repoConfig.projects.map(p => ({
```

Comment:
**question (non-blocking):** When is there more than one matched project that hits this condition? I tried this in a monorepo and couldn't tell what was supposed to be different.

Reply:
It would happen if there were multiple Projects with the same `rootDirectory` setting applied. This could happen for example with Storybook, where there's a Next.js app and the Storybook project with the same root directory. In this case, it's ambiguous which project is intended to be interacted with, so we show the picker.